web/src/main/java/se/inera/intyg/intygstjanst/web/integration/citizen/ListCitizenCertificatesRequestDTO.java

```diff
@@ -0,0 +1,21 @@
+package se.inera.intyg.intygstjanst.web.integration.citizen;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import se.inera.intyg.intygstjanst.web.service.dto.citizen.CitizenCertificateStatusTypeDTO;
+
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ListCitizenCertificatesRequestDTO {
```

Comment:
If you add empty lists as default values the user of the api don't have to define empty lists when calling the api. It makes it easier to use (and test) api.

Now:
`POST http://localhost:8180/inera-certificate/internalapi/citizen
Content-Type: application/json

{
  "patientId": "19121212-1212",
  "years": [],
  "certificateTypes": [],
  "statuses": [],
  "units": []
}`

After:
`POST http://localhost:8180/inera-certificate/internalapi/citizen
Content-Type: application/json

{
  "patientId": "19121212-1212"
}`